### ISSUE TITLE
Please provide docker image for cli

### ISSUE THREAD

**Issue by everflux** [fnproject/cli#317]
This repo contains a Dockerfile, please setup an automated build and provide the cli as docker image so easy setup/experimentation is possible.

**Comment by denismakogon** [comment]
Hi and welcome! It's already as simple as possible to start with the Fn. You can download a binary just from the GitHub, so, don't even need a docker for the CLI.

**Comment by zootalures** [comment]
Iirc we do/did  have a docker image on  dockerhub but I don’t think it’s being automatically updated,  I’ll take a look at making sure it is

**Comment by zootalures** [comment]
Yep, we are build ’fnproject/fn’ on every release with latest as latest version and a version tagged at the appropriate release version 


From. ‘release.sh’ :

‘’’

docker build -t fnproject/fn:latest .
docker tag fnproject/fn:latest fnproject/fn:$version
docker push fnproject/fn:$version
docker push fnproject/fn:latest
‘’’

Will leave this open as it’s not clear from the README.md and it should be

**Comment by zootalures** [comment]
Ack, thanks for raising!

**Comment by josephholsten** [comment]
@zootalures looks like only a project owner can set up automatic builds: https://docs.docker.com/docker-hub/builds/
